conans/util/windows.py

```diff
@@ -48,6 +50,16 @@ def path_shortener(path, short_paths):
         drive = os.path.splitdrive(path)[0]
         short_home = drive + "/.conan"
     mkdir(short_home)
+
+    # Workaround for short_home living in NTFS file systems. Give full control permission to current user to avoid
+    # access problems in cygwing/msys2 windows subsystems when using short_home folder
+    try:
+        cmd = r'cacls %s /E /G "%s\%s":F' % (short_home, os.environ['USERDOMAIN'], os.environ['USERNAME'])
```

Comment:
This has to be done everytime that a new short-paths link is created?
Maybe it would be good to do it only once for CONAN_USER_HOME_SHORT.

Reply:
Why not ``icacls`` instead?

Reply:
>This has to be done everytime that a new short-paths link is created?
Maybe it would be good to do it only once for CONAN_USER_HOME_SHORT.

If the user changes (in my case I was testing with 2 users this issue), it needs to grant priviledges to the current user. IMHO check/assign would be overkilling (and maybe slow)  because that command is ensuring that you are granting the requested priviledge.

>Why not icacls instead?

I read these notes https://en.wikipedia.org/wiki/Cacls and it looks like icalcs has some [serious bug](https://en.wikipedia.org/wiki/Cacls) that I didn't want to introduce for this tiny change.